data/human/culture conversations.txt

```diff
@@ -1101,3 +1101,57 @@ mission "Little Vienna in New Austria"
 			label backtoship
 			`	With the atmosphere steeped in nostalgia for a distant city on a distant world long past, you return to your ship.`
 
+mission "Set Easter Date"
+	landing
+	repeat
+	non-blocking
+	to offer
+		"easter: a" != "year" * 19
```

Comment:
Should this not also be a modulo? Otherwise, this condition is never going to be equal to the year times 19.
```suggestion
		"easter: a" != "year" % 19
```